database/factories/UserFactory.php

```diff
@@ -2,19 +2,24 @@
 
 declare(strict_types=1);
 
+namespace Database\Factories;
+
 use App\User;
-use Faker\Generator as Faker;
-use Illuminate\Database\Eloquent\Factory;
+use Illuminate\Database\Eloquent\Factories\Factory;
 use Illuminate\Support\Str;
 
-/** @var Factory $factory */
+class UserFactory extends Factory
+{
+    protected $model = User::class;
 
-$factory->define(User::class, function (Faker $faker) {
-    return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
-        'email_verified_at' => now(),
-        'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-        'remember_token' => Str::random(10),
-    ];
-});
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->name,
```

Comment:
Please use double quotes.

Reply:
Of course. If you would reconsider using ECS, you can use my fixer for that: https://github.com/krzysztofrewak/php-cs-double-quote-fixer